nixos/modules/services/networking/frr.nix

```diff
@@ -137,6 +136,19 @@ in
             '';
           };
         };
+        mgmt = (serviceOptions "mgmt") // {
+          enable = mkOption {
+            type = types.bool;
+            default = isEnabled "static";
```

Comment:
Please add a `defaultText` that explains this relationship.

Reply:
I see, doing
`defaultText = lib.literalExpression "config.services.frr.static.enable";`

Reply:
Yeah, that would work.

> "this is when staticd is enabled" in description is not sufficient?

Expressions as the default don't render well on search.nixos.org iirc.